src/app/wizard/step/cluster/component.ts

```diff
@@ -186,7 +186,7 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
 
   private _enforce(control: Controls, isEnforced: boolean): void {
     if (isEnforced) {
-      this.form.get(control).setValue(true);
+      this.form.get(control).setValue(this._settings.opaOptions.enabled);
```

Comment:
You might not be done yet, but just wanted to mention (because it came into my mind):
The `_enforce` function is also used in line 124 `this._enforce(Controls.AuditLogging, dc.spec.enforceAuditLogging);`.
AuditLogging is specified via the dc and should definitely be an enabled + disabled.

Also please change the OPA Enforced behavior in the edit cluster dialog.

Reply:
In such a case (enforced by dc), all such settings should follow this behavior. I'll then leave it as is and adjust the test itself. We should update other settings to follow this logic.

Reply:
I have created a follow up issue #3355